### PR TITLE
Corrected MSDOS filesystem wording (angulargui).

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/storage.rst [hunk 1]
```diff
@@ -1788,10 +1788,10 @@ Importing a Disk
 
 The :menuselection:`Pool --> Import Disk` screen, shown in
 :numref:`Figure %s <zfs_import_disk_fig>`, is used to import a
-**single** disk that is formatted with a UFS, NTFS, MSDOS, or EXT2
-filesystem. The import is meant to be a temporary measure to copy the
-data from a disk to an existing ZFS dataset. Only one disk can be
-imported at a time.
+**single** disk that is formatted with UFS (BSD Unix), FAT or
+NTFS (Windows), or EXT2 (Linux) filesystems. The
+import is meant to be a temporary measure to copy the data from a disk
```
Comment (Mrt134) [inline userguide/storage.rst]:
```suggestion
import is temporarily used to copy the data from a disk
```

File: userguide/storage.rst [hunk 2]
```diff
@@ -1788,10 +1788,10 @@ Importing a Disk
 
 The :menuselection:`Pool --> Import Disk` screen, shown in
 :numref:`Figure %s <zfs_import_disk_fig>`, is used to import a
-**single** disk that is formatted with a UFS, NTFS, MSDOS, or EXT2
-filesystem. The import is meant to be a temporary measure to copy the
-data from a disk to an existing ZFS dataset. Only one disk can be
-imported at a time.
+**single** disk that is formatted with UFS (BSD Unix), FAT or
```
Comment (wblock) [inline userguide/storage.rst]: I'm not sure the emphasis on "single" is correct here.  We load the kernel graid module to be able to support importing those, and they are not single disks.

Also, s/FAT/FAT (MSDOS)/

File: userguide/storage.rst (hunk 1 above)
Comment (wblock) [reply userguide/storage.rst]: This whole sentence is unclear.  It's not temporary.  Rather, it's a one-time thing.

`` This is a designed to be used as a one-time import, copying the data from that disk into a dataset on the %brand% system. ``

File: userguide/storage.rst [hunk 3]
```diff
@@ -1788,10 +1788,10 @@ Importing a Disk
...
+NTFS (Windows), or EXT2 (Linux) filesystems. The
+import is meant to be a temporary measure to copy the data from a disk
+to an existing ZFS dataset. Only one disk can be imported at a time.
```
Comment (wblock) [inline userguide/storage.rst]: Again, I'm not sure if the "one disk" thing is truly valid.

Comment (a-stjohn) [reply userguide/storage.rst]: Confirmed that it is one at a time by Mav and Caleb.

**Pull request merged**